SwiftyBluetooth/Source/Central.swift

```diff
@@ -144,6 +144,21 @@ extension Central {
         return self.centralProxy.centralManager.isScanning
     }
     
+    /// The underlying CBCentralManager delegate value
+    public var delegate: CBCentralManagerDelegate? {
```

Comment:
If the CentralProxy's CBCentralManager instance's delegate is not an instance of CentralProxy, the Central shared instance will become unusable. I'd rather not let other people do what you're trying to do, they could unknowningly do it and then wonder why the Central instance is not calling their callbacks.

Perhaps you can tell to me why you want to change the CentralManager delegate and we can come to an alternative solution.